pkg/virt-launcher/monitor_test.go

```diff
@@ -205,12 +204,9 @@ var _ = Describe("VirtLauncher", func() {
 					done <- "exit"
 				}()
 
-				time.Sleep(time.Second)
-				Expect(gracefulShutdownCallbackTriggered).To(BeFalse())
+				Consistently(gracefulShutdownChannel).Should(Not(BeClosed()))
```

Comment:
This is using polling and changes the timeout from 1sec to 100msec.
How about using events/signals instead? E.g a timer with a select.

It may be an overkill, but I dislike the use of the polling too much to avoid commenting about it. :)

Reply:
`Consistently` and `Eventually` are the way to go. `selects` would only help with `Eventually` and not with `Consistently`, so we would end up again with `Sleep` which does not mean much. The gomega language transports pretty clearly what one wants to test.